### PR TITLE
Improve pod log

### PR DESCRIPTION
**Pull request by dtaniwaki**
I made `idleTimeout` configurable for the `getPodLogSource` method because pod logs sometimes become idle of longer than 60 seconds which is [the default of `akka-http`](https://doc.akka.io/docs/akka-http/current/common/timeouts.html#common-timeouts).

### CODE REVIEW

**Comment by dtaniwaki** [comment]
@doriordan Is there any chance to have this feature in the next version?

File: client/src/main/scala/skuber/api/package.scala [hunk 1]
```diff
@@ -226,13 +227,22 @@ package object client {
...
+     private[skuber] def invokeLog(request: HttpRequest)(implicit lc: LoggingContext): Future[HttpResponse] = invoke(request, podLogSettings)
+
+     private[skuber] def invoke(request: HttpRequest, settings: ConnectionPoolSettings = ConnectionPoolSettings(actorSystem))(implicit lc: LoggingContext): Future[HttpResponse] = {
        if (isClosed) {
          logError("Attempt was made to invoke request on closed API request context")
          throw new IllegalStateException("Request context has been closed")
        }
        logInfo(logConfig.logRequestBasic, s"about to send HTTP request: ${request.method.value} ${request.uri.toString}")
-       val responseFut = requestInvoker(request, watch)
+       val connectionContext = sslContext
```
Comment (doriordan) [inline client/src/main/scala/skuber/api/package.scala]: This creates a new connection context on every request (in SSL case), can we create the connection context once and reuse as before?

**Comment by doriordan** [comment]
@dtaniwaki thanks for this

Just one comment from an initial review.

My current plan is to release `2.0.12` in the next couple of weeks, it should be possible to get this into that.

Just for information - I have been refactoring the `skuber.api` class for an intended `2.1.0` release on a separate branch (those changes are part of https://github.com/doriordan/skuber/tree/2.1-preview at the moment)

As part of this the RequestContext code has been moved to a new implementation class (but keeping the application API backwards compatible). 

Once `2.0.12` is released I plan to immediately merge the refactored `skuber.api` package into master, so effectively `2.0.12` is the final 2.0 release. 

At that stage any recent conflicting changes to `skuber.api` will need to be merged which complicates the merge a bit - so to keep that from becoming a bigger issue this may be the last PR that impacts the api package that will be merged before `2.1.0`.

**Comment by dtaniwaki** [comment]
@doriordan I have fixed the code based on your feedback. Would you check it again?

**Comment by doriordan** [comment]
LGTM thanks

**Pull request merged**